developer/upgrading/0.26.0.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Upgrading to 0.26.0
+---
+
+If you haven't yet upgraded to `0.25.0`, please review the [0.25.0 upgrade
+guide](/developer/upgrading/0.25.0) first.
+
+`v0.26.0` of the runtime optimizes the client bundle for Makeswift integrated
+hosts, and reorganizes several package exports. Part of this reorganization
+includes moving several internal functions and types to internal-only entry
```

Comment:
As part of this reorganization, several internal functions and types have been moved to internal-only entry points.